vision_agent_tools/tools/owlv2.py

```diff
@@ -82,7 +82,9 @@ def __call__(
         """
         texts = [prompts]
         # Run model inference here
-        inputs = self._processor(text=texts, images=image, return_tensors="pt")
+        inputs = self._processor(text=texts, images=image, return_tensors="pt").to(
+            self.device
+        )
```

Comment:
why not do this on the __init__?

Reply:
@camiloaz it throws an error saying that `self._processor` does not have `to()` method.

Reply:
If you try to do something similar to the thing that we do with the model like `self. _processor.to(self.device)` you will get the error. You have to actually call the processor in order to apply the `to()` method.